Project-Aurora/Project-Aurora/Settings/Layers/AmbilightLayerHandler.cs

```diff
@@ -87,6 +87,26 @@ public class AmbilightLayerHandlerProperties : LayerHandlerProperties2Color<Ambi
         [JsonIgnore]
         public String SpecificProcess { get { return Logic._SpecificProcess ?? _SpecificProcess ?? String.Empty; } }
 
+        public int? _CoordinateX { get; set; }
```

Comment:
You could probably replace all these coordinate variables with the Rectangle type